packages/hermes-test/app/(tabs)/_layout.tsx

```
@@ -0,0 +1,46 @@
+import { Tabs } from "expo-router";
+import React from "react";
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.

(lint/correctness/noUnusedImports)